Dockerfile

```diff
@@ -30,8 +30,14 @@ RUN go generate && CGO_ENABLED=0 GOOS=linux go build -o /usr/bin/spark-operator
 
 FROM ${SPARK_IMAGE}
 COPY --from=builder /usr/bin/spark-operator /usr/bin/
-RUN apk add --no-cache openssl curl tini
+RUN apt-get update \
+    && apt-get install -y openssl curl tini \
+    && rm -rf /var/lib/apt/lists/*
 COPY hack/gencerts.sh /usr/bin/
+ENV TINI_VERSION v0.18.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
+RUN chmod +x /tini \
+    && mv /tini /sbin/tini
```

Comment:
`tini` is being installed through apt-get and the Debian Buster base distro already includes tini v0.18.0. These lines could be removed and `entrypoint.sh` fixed up to use the `/usr/bin/tini` path provided by the Debian tini package

Reply:
Thanks.